.github/renovate.json5

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval"
+    ],
+    "dependencyDashboard": true,
+    "dependencyDashboardLabels": ["dependencies", "repo maintenance"],
+    "enabledManagers": ["github-actions", "npm"],
```

Comment:
Why are you only allowing the `github-actions` and `npm` managers? Usually I would recommend that you allow Renovate to update all the dependencies that it can find.

Reply:
My context coming in was to enable only the bits desired. Since the repo mainly just those, I figured it made sense to just enable for them. But I can just remove it - makes sense. Thanks!